<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#headerLabel {
    color: white;
    background-color: #0b3d91;
    font-size: 24px;
    font-weight: bold;
    padding: 10px;
    border: 2px solid #ffffff;
    border-radius: 10px;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget {
    background-image: url(:/images/heic0206a.jpg);
    background-position: center;
    background-repeat: no-repeat;
}
</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>801</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
    background-color: rgba(0, 0, 0, 0.7);  /* –ü–æ–ª—É–ø—Ä–æ–∑—Ä–∞—á–Ω—ã–π —á–µ—Ä–Ω—ã–π —Ñ–æ–Ω */
    color: white;  /* –ë–µ–ª—ã–π —Ü–≤–µ—Ç —Ç–µ–∫—Å—Ç–∞ */
    gridline-color: #ffffff;  /* –ë–µ–ª—ã–µ –ª–∏–Ω–∏–∏ —Å–µ—Ç–∫–∏ */
    font-size: 14px;  /* –†–∞–∑–º–µ—Ä —à—Ä–∏—Ñ—Ç–∞ */
    border: 2px solid #ffffff;  /* –ë–µ–ª–∞—è –≥—Ä–∞–Ω–∏—Ü–∞ */
    border-radius: 10px;  /* –°–∫—Ä—É–≥–ª–µ–Ω–Ω—ã–µ —É–≥–ª—ã */
}

QHeaderView::section {
    background-color: #001f3f;  /* –¢–µ–º–Ω–æ-—Å–∏–Ω–∏–π —Ü–≤–µ—Ç –∑–∞–≥–æ–ª–æ–≤–∫–æ–≤ */
    color: white;  /* –ë–µ–ª—ã–π —Ü–≤–µ—Ç —Ç–µ–∫—Å—Ç–∞ –≤ –∑–∞–≥–æ–ª–æ–≤–∫–∞—Ö */
    padding: 5px;  /* –í–Ω—É—Ç—Ä–µ–Ω–Ω–∏–π –æ—Ç—Å—Ç—É–ø */
    border: 1px solid #ffffff;  /* –ë–µ–ª–∞—è –≥—Ä–∞–Ω–∏—Ü–∞ */
}

QTableCornerButton::section {
    background-color: #001f3f;  /* –¢–µ–º–Ω–æ-—Å–∏–Ω–∏–π —Ü–≤–µ—Ç —É–≥–ª–æ–≤–æ–π –∫–Ω–æ–ø–∫–∏ */
    border: 1px solid #ffffff;  /* –ë–µ–ª–∞—è –≥—Ä–∞–Ω–∏—Ü–∞ */
}

QTableWidget::item {
    padding: 5px;  /* –í–Ω—É—Ç—Ä–µ–Ω–Ω–∏–π –æ—Ç—Å—Ç—É–ø */
    border: none;  /* –£–±—Ä–∞—Ç—å –≥—Ä–∞–Ω–∏—Ü—ã —è—á–µ–µ–∫ */
    background-color: transparent;  /* –ü—Ä–æ–∑—Ä–∞—á–Ω—ã–π —Ñ–æ–Ω —è—á–µ–µ–∫ */
}

QTableWidget::item:selected {
    background-color: #0074D9;  /* –¶–≤–µ—Ç –≤—ã–¥–µ–ª–µ–Ω–∏—è */
    color: #ffffff;  /* –¶–≤–µ—Ç —Ç–µ–∫—Å—Ç–∞ –ø—Ä–∏ –≤—ã–¥–µ–ª–µ–Ω–∏–∏ */
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="headerLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>981</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#headerLabel {
    color: white;  /* –ë–µ–ª—ã–π —Ü–≤–µ—Ç —Ç–µ–∫—Å—Ç–∞ */
    background-color: rgba(0, 0, 0, 0.7);  /* –ü–æ–ª—É–ø—Ä–æ–∑—Ä–∞—á–Ω—ã–π —á–µ—Ä–Ω—ã–π —Ñ–æ–Ω */
    font-size: 24px;  /* –†–∞–∑–º–µ—Ä —à—Ä–∏—Ñ—Ç–∞ */
    font-weight: bold;  /* –ñ–∏—Ä–Ω—ã–π —à—Ä–∏—Ñ—Ç */
    padding: 10px;  /* –í–Ω—É—Ç—Ä–µ–Ω–Ω–∏–π –æ—Ç—Å—Ç—É–ø */
    border: 2px solid #ffffff;  /* –ë–µ–ª–∞—è –≥—Ä–∞–Ω–∏—Ü–∞ */
    border-radius: 10px;  /* –°–∫—Ä—É–≥–ª–µ–Ω–Ω—ã–µ —É–≥–ª—ã */
    text-align: center;  /* –¶–µ–Ω—Ç—Ä–∏—Ä–æ–≤–∞–Ω–∏–µ —Ç–µ–∫—Å—Ç–∞ */
}
</string>
    </property>
    <property name="text">
     <string>NASA Near Earth Objects ‚Äß‚ÇäÀö ‚òÅÔ∏è‚ãÖ‚ô°ü™ê‡ºò‚ãÜ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="clusterIdInput">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#clusterIdInput {
    color: white;  /* –ë–µ–ª—ã–π —Ü–≤–µ—Ç —Ç–µ–∫—Å—Ç–∞ */
    background-color: rgba(0, 0, 0, 0.7);  /* –ü–æ–ª—É–ø—Ä–æ–∑—Ä–∞—á–Ω—ã–π —á–µ—Ä–Ω—ã–π —Ñ–æ–Ω */
    font-size: 14px;  /* –†–∞–∑–º–µ—Ä —à—Ä–∏—Ñ—Ç–∞ */
    font-weight: bold;  /* –ñ–∏—Ä–Ω—ã–π —à—Ä–∏—Ñ—Ç */
    padding: 10px;  /* –í–Ω—É—Ç—Ä–µ–Ω–Ω–∏–π –æ—Ç—Å—Ç—É–ø */
    border: 2px solid #ffffff;  /* –ë–µ–ª–∞—è –≥—Ä–∞–Ω–∏—Ü–∞ */
    border-radius: 10px;  /* –°–∫—Ä—É–≥–ª–µ–Ω–Ω—ã–µ —É–≥–ª—ã */
    text-align: center;  /* –¶–µ–Ω—Ç—Ä–∏—Ä–æ–≤–∞–Ω–∏–µ —Ç–µ–∫—Å—Ç–∞ */
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchClusterButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>110</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>üîé</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>160</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="fetchDataButton">
       <property name="styleSheet">
        <string notr="true">QPushButton#fetchDataButton {
    color: white;
    background-color: rgba(0, 0, 0, 0.7);
    font-size: 14px;
    font-weight: bold;
    padding: 10px;
    border: 2px solid #ffffff;
    border-radius: 10px;
    text-align: center;
}
</string>
       </property>
       <property name="text">
        <string>Fetch Data üéÄ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clusterButton">
       <property name="styleSheet">
        <string notr="true">QPushButton#clusterButton {
    color: white;
    background-color: rgba(0, 0, 0, 0.7);
    font-size: 14px;
    font-weight: bold;
    padding: 10px;
    border: 2px solid #ffffff;
    border-radius: 10px;
    text-align: center;
}


</string>
       </property>
       <property name="text">
        <string>Run Clustering üéÄ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="restoreTableButton">
       <property name="styleSheet">
        <string notr="true">QPushButton#restoreTableButton {
    color: white;
    background-color: rgba(0, 0, 0, 0.7);
    font-size: 14px;
    font-weight: bold;
    padding: 10px;
    border: 2px solid #ffffff;
    border-radius: 10px;
    text-align: center;
}
</string>
       </property>
       <property name="text">
        <string>Restore Table üéÄ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearTableButton">
       <property name="styleSheet">
        <string notr="true">QPushButton#clearTableButton {
    color: white;
    background-color: rgba(0, 0, 0, 0.7);
    font-size: 14px;
    font-weight: bold;
    padding: 10px;
    border: 2px solid #ffffff;
    border-radius: 10px;
    text-align: center;
}
</string>
       </property>
       <property name="text">
        <string>Clear Table üéÄ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label {
    color: white;
    font-size: 20px;
    font-weight: bold;
    text-align: center;
}</string>
    </property>
    <property name="text">
     <string>Search for particular Cluster:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1005</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
